abstractions/php/src/Serialization/ParseNode.php

```diff
@@ -44,16 +44,16 @@ public function getFloatValue(): ?float;
 
     /**
      * Gets the model object value of the node.
-     * @param string $type The type for the Parsable object.
+     * @param array{string,string} $type The type for the Parsable object.
      * @return Parsable|null the model object value of the node.
      */
-    public function getObjectValue(string $type): ?Parsable;
+    public function getObjectValue(array $type): ?Parsable;
```

Comment:
I'm curious to why an array is needed here since we're only ever going to pass a single type?
And actually it should be a callback/method reference so the discriminator can be handled as well.

Reply:
Hello @baywet thanks for the review. The array is used as a way to represent a method reference since there is no way to assign a method reference to a variable without actually calling it. [This](https://stackoverflow.com/a/26436247) might be helpful to understand why I used the array.

Reply:
thanks for sharing the context!